Web Page Design

LoboArkano opened the pull request:
In this pull request, we do a Youtube clone page.

LoboArkano committed:
Update README.md

Add template

Commit:
Created HTML Basic Structure

Commit:
Added png images

Commit:
Created nav elements

RossielCS committed:
Added the img elements

Added new div elements which contains images for the video thumbnails.
This will be for the menu on the right of the webpage.

Commit:
Change size of navbar elements

RossielCS committed:
Modified properties of the video_list class

Changed the size and position of the images and text f the right menu.

Commit:
Change size of nav bar elements and replacing images

RossielCS committed:
Added the description of the main video

Added two p elements with the description of the video and also the upload date.
Added a text-align property.

RossielCS committed:
Deleted elements from the index

Deleted elements from de head tag,
also the nav_container, search_bar, principal_content and video_text.
We decided to start the project again.

RossielCS committed:
Deleted properties from th stylesheet

Deleted all the properties from the style.css and added properties for the body, navbar, container.

Commit:
Add navbar elemets

RossielCS committed:
Added main description and contents for the sidebar

Added a p tag with the description of the main video.
Also, added tue top for the sidebar and the thumbnails.

RossielCS committed:
Added properties for the main video description and sidebar

Added properties for the components of the main video description.
Also, properties for the sidebar elementes.

Commit:
Add title and info of the channel

Commit:
Add style for main_v_title

RossielCS committed:
Added pictures and made modifications in the stylesheet

Added pictures in the images folder for the icons and thumbnails.
Made modifications for the properties of  body, container and main video.

Commit:
Add images for share, like and dislike butttons

Commit:
Change size of the video

Commit:
Validate HTML markup

Commit:
Update HTML tags

RossielCS committed:
Added information about the project in the README file.

We added information about the project along with a screenshot, the prerequisites for it, the GitHub and LinkedIn of the authors.

Commit:
Update README.md

Commit:
Fix typo on README.md

pelzolga123 reviewed (changes requested):
Project requires changes.

pelzolga123 commented:
## Hi Team! ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
### Your project looks :+1: amazing!

Here are the changes you need to make, before this project can be approved:
* [x] check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows one error in your project.
* [x] Pull Request should have meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
#### Navigation bar
* [x] Project should have a`<button>` tag that represents the search button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)) Right now it is just an image.
* [x] Project should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
#### Video section
* [x] Project should have like and dislike icons ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)) Right now you have images there.
 * [x] `Share` should be a button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [x] `Subscribe` should be a button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

RossielCS committed:
Added stickler and styleint files to the root

Added stickler and styleint files to the root of the projects for the source code analasys.
Fixed the errors noted by the stylelint.

Commit:
Add icon to the navbar

RossielCS committed:
Added icons and class names

Changed the like and dislike img for icon.
Added the video_img class for the images in the sidebar class.
Added new buttons for subscribe and share.

RossielCS committed:
Fixed errors noted by stickler in the stylesheet

Changed the line-height for the main_v_title class.
Also, deleted a white space on line 263.

kobiyoyo reviewed (changes requested):
## Hi Team      ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
#### Great Job so far, but there are a few more things required. 
### General
- [x] Pull request should have a meaning short description.
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.

## Video section
 - [x] The share should be a button (screenshot from Pathwright)
-  [x] Subscribe should be a button with styled text not image (screenshot from Pathwright)

### Make changes and submit another code review request.
*Happy coding*

kobiyoyo commented on index.html:
```diff
@@ -0,0 +1,278 @@
+				</div>
+				<div id="name_channel">freeCodeCamp.org</div>
+				<div id="subscribe_b">
+					<button style="border: none">
```
no inline CSS

kobiyoyo commented on index.html:
```diff
@@ -0,0 +1,278 @@
+				<div id="subscribers">1,67M</div>
+				<div id="views">150,384 views</div>
+				<div id="share">
+					<button style="border: none">
```
no inline CSS

RossielCS committed:
Deleted the in line style of the buttons

Deleted the in line style of the buttons and added the properties to the stylesheet.

Commit:
Replace button subscribe image with css

RossielCS committed:
Added class names to the buttons of the main video class

Added the class main_button to the buttons of the main video.
Fixed some errors that styleint found in the stylesheet.

RossielCS committed:
Added text alternatives to images

Added text alternatives to all the images that had ther property empty.

mosesogwo reviewed (approved):
**Hi team. Good job making the requested changes:+1:. Your project is approved :heavy_check_mark:**.

You can now merge to master.

**Good luck in your next project:stars:**
